src/connector.js

```diff
@@ -44,6 +44,7 @@ let paramsDetect = {};
 let params = {};
 let urlParams;
 let hashParams;
+let originLevel3RelativeUrl;
```

Comment:
Because of the try catch, if the URL is invalid, the originLevel3RelativeUrl would never get a value. Therefore, I think it would be best to have it declared as an empty string by default:
```suggestion
let originLevel3RelativeUrl = "";
```